Client/mods/deathmatch/logic/luadefs/CLuaVehicleDefs.cpp

```diff
@@ -4273,3 +4275,29 @@ std::variant<bool, std::array<CVector, 4>> CLuaVehicleDefs::OOP_GetVehicleEntryP
 
     return entryPoints;
 }
+
+bool CLuaVehicleDefs::AddVehicleSirens(CClientVehicle* vehicle, std::uint8_t sirenType,  std::uint8_t sirenCount, std::optional<bool> enable360, std::optional<bool> enableLOSCheck, std::optional<bool> enableRandomiser, std::optional<bool> enableSilent)
+{
+    eClientVehicleType vehicleType = vehicle->GetVehicleType();
+
+    if (vehicleType == CLIENTVEHICLE_CAR)
+    {
+        if (sirenType >= 1 && sirenType <= 6)
+        {
+            if (sirenCount <= SIREN_COUNT_MAX)
+            {
+                vehicle->GiveVehicleSirens(sirenType, sirenCount);
+                vehicle->SetVehicleFlags(enable360.value_or(false), enableRandomiser.value_or(true), enableLOSCheck.value_or(true), enableSilent.value_or(false));
+                return true;
+            }
+        }
+    }
```

Comment:
I think you could use early-return here to improve readability. Apart from that everything seems fine